src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js

```diff
@@ -295,19 +313,19 @@
 				label: "",
 				parameters: []
 			});
-    }
+        }
```

Comment:
Can you clean the formatting up a little, this leaves an ugly indent.

Reply:
I'm having a bit of trouble seeing how to fix this. It looks clean in my IDE.

Reply:
Hmm.. I opened `pipeline.launch.js` in emacs and the indenting looks a lot different than it does in IntelliJ Idea or VS Code. I don't have the 'Ultimate Edition' of IntelliJ so it doesn't have greate JS support.

I've gone through the whole file and tried to give it a consistent 4-space indentation throughout. Hope that doesn't make the diff too busy to read.

Reply:
We are using [Prettier](https://prettier.io/) for formatting on all our new files.  If you are set on emacs there is an [emacs minor mode](https://github.com/prettier/prettier-emacs).  I personally use IntelliJ ultimate, but I have used (and Tom does use) VSCode and it is a fantastic front-end editor.  We also use eslint.

It is very difficult to see the changes you made now that you did format the entire file.  If yo do make updates to what I suggested can you indicate the line number or someway for me to know where to go.